### PR TITLE
Backport to 2.0.15.1: Async Package load

### PR DESCRIPTION
**Pull request by shana**
This is #528 applied to 2.0.15.1. We may need to do a release for it before master is ready, and I don't want to lose the conversation on #528.

### CODE REVIEW

File: src/GitHub.VisualStudio/GitHubPackage.cs [hunk 1]
```diff
@@ -112,31 +154,36 @@ Version VSVersion
...
-                var shell = await GetServiceAsync(typeof(SVsShell)) as IVsShell;
-                IVsPackage startPagePackage;
-                if (ErrorHandler.Failed(shell?.LoadPackage(new Guid(StartPagePreview4PackageId), out startPagePackage) ?? -1))
-                {
-                    // ¯\_(ツ)_/¯
-                }
-            }
+            AddService(typeof(IUsageTracker), CreateService, true);
+            AddService(typeof(IMenuProvider), CreateService, true);
+            return Task.FromResult<object>(null);
```
Comment (tocsoft) [inline src/GitHub.VisualStudio/GitHubPackage.cs]: I feel ``Task.FromResult(true)`` is slightly easier to read than  `Task.FromResult<object>(null)`. Less brackets to track while mentally parsing.

**Comment by tocsoft** [comment]
@shana you will probably want to back port #621 (for version numbers) into this as well, and possibly #651 (for broken icon) if you are pushing out a release not based on master.

File: src/GitHub.VisualStudio/GitHubPackage.cs (hunk 1 above)
Comment (shana) [inline src/GitHub.VisualStudio/GitHubPackage.cs]: We have `Task.CompletedTask` for this purpose, let's use that.

**Comment by shana** [comment]
@tocsoft Yes, there are several other fixes we'll need, we're still trying to figure out if those go in this release or separately.

**Comment by tocsoft** [comment]
👍

**Pull request merged**